fuzz/.gitignore

```diff
@@ -0,0 +1,4 @@
+
+target
+corpus
```

Comment:
Use the gitignore file in root directory instead.

Reply:
It's automatically generated.

Reply:
So will it be regenerated again?